xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/diff_macros.vm

```diff
@@ -292,11 +292,11 @@
       #set ($classReference = $services.model.resolveDocument($className))
       #set ($classPropertyReference = $services.model.createEntityReference($propertyDiff.propName, 'CLASS_PROPERTY',
         $classReference))
-      <dt id="#diffAnchor($classPropertyReference)">
+      <dt id="#diffAnchor($classPropertyReference)" #diffDataReference($classPropertyReference)>
         #diffIcon('cube' $propertyDiff.action)
         $propertyDiff.propName
         <span class="hint">
-          ##($escapetool.xml($className))
+          ($escapetool.xml($className))
```

Comment:
I don't like this either. If need be, we could improve the separation between page properties, object properties and class properties. I'm fine to display the reference on hover (using the ``title`` attribute), but I don't think we should repeat the class reference after each property name.